raphael-cli/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "raphael-cli"
+version = "0.1.0"
+edition = "2021"
+default-run = "raphael-cli"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+simulator = { path = "../simulator" }
+solvers = { path = "../solvers" }
+game-data = { path = "../game_data" }
+
+clap = { version = "4.4.11", features = ["derive", "wrap_help"] }
+
+[[bin]]
```

Comment:
The `[[bin]]` section can be removed.
Cargo will automatically find the entry point at `src/main.rs` and the binary's name will be the same as the crate's name.

Reply:
Hmm, at some point I got an error about no `[lib]` or `[[bin]]` being present, so I added it. Something else must've been off because removing it from `Cargo.toml` does what you say!